src/Grpc/test/InteropTests/Helpers/ClientProcess.cs

```diff
@@ -15,9 +16,13 @@ public class ClientProcess : IDisposable
         private readonly Process _process;
         private readonly ProcessEx _processEx;
         private readonly TaskCompletionSource<object> _startTcs;
+        private readonly StringBuilder _output;
```

Comment:
Suggest cleaning this object up in `Dispose()`

Reply:
It will get GCed with ClientProcess so I'm not worried about it